File: AttendanceMachine/lib/models/attendance_result.dart
class AttendanceResult {
  final String deviceId;
  final String name;
  final String employeeCode;
  final String time;
  final String status;

  AttendanceResult({
    required this.deviceId,
    required this.name,
    required this.employeeCode,
    required this.time,
    required this.status,
  });

  factory AttendanceResult.fromJson(Map<dynamic, dynamic> json) {
    return AttendanceResult(
      deviceId: json['deviceId'],
      name: json['name'],
      employeeCode: json['employeeCode'],
      time: json['time'],
      status: json['status'],
    );
  }
}

File: ServerIoT/src/main/java/com/example/demo/Models/Response/AttendanceResult.java
package com.example.demo.Models.Response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AttendanceResult {
    private String deviceId;
    private String name;
    private String employeeCode;
    private String time;
    private String status;
}
